src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManagerTest.java

```diff
@@ -155,4 +155,31 @@ void generateRandomQuestions_appliesExclusionFilterForDeprecatedQuestions() thro
     assertEquals(Constants.BooleanOperator.NOT, deprecatedFilter.getOperator());
     assertEquals(Collections.singletonList("true"), deprecatedFilter.getValues());
   }
+
+  @Test
+  void generateRandomQuestions_appliesExclusionFilterForRegressionTest() throws
+      ContentManagerException {
+
+    Capture<List<BooleanSearchClause>> capturedFilters = Capture.newInstance();
+    expect(dummyContentManager.findByFieldNamesRandomOrder(
+        capture(capturedFilters),
+        anyInt(),
+        anyInt(),
+        anyLong())
+    ).andStubReturn(new ResultsWrapper<>());
+    replay(dummyContentManager);
+
+    // Act
+    gameManager.generateRandomQuestions(new GameFilter(), 5);
+
+    // Assert
+    // check that one of the filters sent to GitContentManager was the regression test exclusion filter
+    List<BooleanSearchClause> filters = capturedFilters.getValues().get(0);
+    BooleanSearchClause tagsFilter = filters.stream()
+        .filter(f -> Objects.equals(f.getField(), "tags")).collect(Collectors.toList()).get(0);
```

Comment:
You can simplify `collect(Collectors.toList())` by replacing it with `toList()`.

Reply:
Alternatively, you can use `.findFirst().get()` (or similar) instead of having to collect and `get(0)`.

Reply:
And another thing that would make this a bit more clear is to replace the filter parameter with: `f -> f.getField().equals("tags")`